examples/a_new_hope.py

```diff
@@ -34,7 +34,7 @@ def grey_color_func(word, font_size, position, orientation, random_state=None, *
 text = text.replace("LUKE'S", "Luke")
 
 # adding movie script specific stopwords
-stopwords = STOPWORDS.copy()
```

Comment:
That works. I think it's just clearer in the example to explicitly show the reader that stopwords is a set.